maestro-android/src/main/java/dev/mobile/maestro/handlers/AbstractSettingHandler.kt

```diff
@@ -0,0 +1,42 @@
+package dev.mobile.maestro.handlers
+
+import android.content.Context
+import android.content.pm.PackageManager
+import android.util.Log
+
+abstract class AbstractSettingHandler(private val context: Context, private val permissions: List<String>) {
+    fun enable(): Boolean {
```

Comment:
Lets include this code in LocationSettingsHandler itself nobody else is using it.